api/logger.py

```diff
@@ -27,9 +27,7 @@ def warning(self, msg, extra=None):
     self.logger.warning(self._log_output(msg, extra))   
     
   def error(self, msg, extra=None):
-    self.logger.error(self._log_output(msg, extra))
+    self.logger.error(self._log_output(msg, extra), exc_info=True)
```

Comment:
nitpick(non-blocking, super small): since this would log 'NoneType: None' every type an exception was not previously raised, this could be edited to print Exception info only if an exception was actually there.
```suggestion
    self.logger.error(self._log_output(msg, extra), exc_info=self.__is_exception_caught())
    
    def __is_exception_caught(self):
      return sys.exc_info() != (None, None, None)
```